troncos/traces/dd_exporter.py

```diff
@@ -9,17 +19,30 @@
 )
 
 
-class OTLPSpanExporterDD(trace_exporter.OTLPSpanExporter):
+def fix_span(sdk_span: ReadableSpan) -> None:
+    sdk_span._instrumentation_scope = InstrumentationScope(
+        name=sdk_span.name,
+        version="0.0.0",
+    )
+    sdk_span._instrumentation_info = InstrumentationInfo(
+        name=sdk_span.name,
+        version="0.0.0",
+    )
+    if "resource" in sdk_span.attributes:  # type: ignore[operator]
+        sdk_span._name = sdk_span.attributes["resource"]  # type: ignore # noqa: E501
+
+
+class OTLPGrpcSpanExporterDD(grpc_trace_exporter.OTLPSpanExporter if grpc_trace_exporter else object):  # type: ignore  # noqa: E501
```

Comment:
yeah, idk.. awesome parent class :see_no_evil: it does work though!

Reply:
Not the prettiest thing no, but these are internals that we can change later if we want,